### PR TITLE
VULN-3708 - [bucko] got have vulnerabilities

### PR DESCRIPTION
**Pull request by adinaiordache2017**
VULN-3708 - [bucko] got have vulnerabilities

### CODE REVIEW

**Comment by adinaiordache2017** [comment]
docker push

File: package.json [hunk 1]
```diff
@@ -45,6 +45,8 @@
 	"overrides": {
 		"ngrok": {
 			"jsprim": "1.4.2"
-		}
```
Comment (esmailpour-hosein) [inline package.json:48]: Is this still needed after `ngrok` update?

Comment (adinaiordache2017) [reply package.json:48]: No, you're right, that dependency should no longer be there. Thanks!

File: package.json [hunk 2]
```diff
@@ -30,9 +30,10 @@
 		"koa-router": "8.0.8",
 		"koa-session": "5.13.1",
 		"koa-views": "6.2.1",
-		"ngrok": "3.2.7",
+		"ngrok": "4.3.1",
 		"object-hash": "2.0.3",
 		"passport-oauth2": "1.6.1",
+		"request": "2.88.2",
```
[the remark this answers is not in the document]
Comment (adinaiordache2017) [reply package.json:36]: Because it was removed before and the app was broken :-( It has to be a dependency

**Review by esmailpour-hosein — APPROVED** [review]

**Pull request merged**